src/pages/Catalog.tsx

```diff
@@ -238,6 +239,25 @@ const Catalog = () => {
                             </Card>
                         );
                     })}
+                    {visibleAnnouncements < announcements.length && (
+                        <Box
+                            sx={{
+                                width: "100%",
+                                display: "flex",
+                                justifyContent: "center",
+                                marginTop: "10px",
+                            }}
+                        >
+                            <Button
+                                variant="contained"
+                                onClick={() =>
+                                    setVisibleAnnouncements((prev) => prev + 2)
```

Comment:
I agree that 3 is a nice number, perhaps for consistency we keep it to 3 per iteration